internal/service/dns_config/api_auth_zone_resource.go

```diff
@@ -78,6 +78,7 @@ func (r *AuthZoneResource) Create(ctx context.Context, req resource.CreateReques
 		AuthZoneAPI.
 		AuthZoneCreate(ctx).
 		Body(*data.Expand(ctx, &resp.Diagnostics, true)).
+		Inherit("full").
```

Comment:
I would suggest to use a constant for this

Reply:
Where do you think this constant be defined: here or in go-sdk? 
go-sdk would be better IMO

Reply:
Fixed

Reply:
We should eventually move it to go-sdk by declaring at as an enum in the swagger json. For now a constant locally is fine.